plugins/ibm_qradar/unit_test/test_get_ariel_search_by_id.py

```diff
@@ -0,0 +1,81 @@
+    """Unit Test class for Test cases of action : get ariel search by id."""
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Set up an action for test."""
+        cls.action = ArielSearchHelper.default_connector(GetArielSearchById())
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id(self, make_request):
+        """To Test the get serial search by id."""
+        action_params = {"search_id": "search_id"}
+        results = self.action.run(action_params)
+        self.assertEqual(results.get("data")["cursor_id"], "test_cursor_id")
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id_with_poll_interval(self, make_request):
+        """To Test the get serial search by id."""
+        action_params = {"search_id": "search_id", "poll_interval": 1}
+        results = self.action.run(action_params)
+        self.assertEqual(results.get("data")["cursor_id"], "test_cursor_id")
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_wrong_id(self, make_request):
+        """To Test the get serial search by id."""
+        action_params = {"search_id": "wrong"}
+        with self.assertRaises(PluginException):
+            self.action.run(action_params)
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id_wrong_host_url(self, make_request):
+        """To test the get ariel search with wrong host_url.
+
+        :return: None
+        """
+        action_params = {"search_id": "search_id"}
+
+        action = ArielSearchHelper.default_connector(
+            GetArielSearchById(),
+            {
+                "host_url": "http://wrong",
+                "credentials": {"username": "user1", "password": "password"},
+            },
+        )
+
+        with self.assertRaises(PluginException):
+            action.run(action_params)
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id_with_internal_server_error(self, make_request):
+        """To Test the get serial search by ID with internalServerError."""
+        action_params = {"search_id": "internalServerError"}
+        with self.assertRaises(PluginException):
+            self.action.run(action_params)
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id_with_forbidden(self, make_request):
+        """To Test the get serial search by ID with for bidden."""
+        action_params = {"search_id": "checkforbidden"}
+        with self.assertRaises(PluginException):
+            self.action.run(action_params)
+
+    @patch("requests.get", side_effect=ArielSearchHelper.mock_request)
+    def test_get_ariel_search_by_id_with_rate_limit(self, make_request):
+        """To Test the get serial search by ID with for bidden."""
```

Comment:
```suggestion
        """To test the get ariel search by ID with forbidden."""
```
Please double-check the other unit test files for spelling errors, for example please replace `serial search` -> `ariel search` and `for bidden` -> `forbidden` and `To Test the` -> `To test the`